### PR TITLE
Add evicted_nonzero and reclaimed to evictions metrics.

### PR DESCRIPTION
**Pull request by ando-masaki**
I want to this plugin wil be simmilar to memcached_multi_evictions.

See:
https://github.com/mhwest13/Memcached-Munin-Plugin/blob/master/memcached_multi_#L409-L441

### CODE REVIEW

File: mackerel-plugin-memcached/memcached_test.go [hunk 1]
```diff
@@ -54,7 +54,9 @@ STAT conn_yields 1487476
 STAT bytes 621371972
 STAT curr_items 955652
 STAT total_items 2423543841
+STAT evicted_nonzero 10
```
Comment (astj) [inline mackerel-plugin-memcached/memcached_test.go]: memcached's `stats` command doesn't output `evicted_nonzero` stats, according to memcached protocol document: https://github.com/memcached/memcached/blob/master/doc/protocol.txt

`evicted_nonzero` stats is only available as `evicted:{slab}:evicted_nonzero` from `stats items` command.

Comment (astj) [inline mackerel-plugin-memcached/memcached_test.go]: To obtain `evicted_nonzero` like Munin's memcached_multi_, at first the plugin should fetch output from `stat items`, and calculate the sum of each values for key `items:{slab}:evicted_nonzero`. (Munin's memcached_multi_ plugin does it at https://github.com/mhwest13/Memcached-Munin-Plugin/blob/master/memcached_multi_#L936-L941)

File: mackerel-plugin-memcached/memcached.go [hunk 2]
```diff
@@ -109,14 +109,15 @@ func (m MemcachedPlugin) GraphDefinition() map[string](mp.Graphs) {
...
+				mp.Metrics{Name: "evicted_nonzero", Label: "Evictions prior to Expire", Diff: true, Type: "uint64"},
+				mp.Metrics{Name: "reclaimed", Label: "Reclaimed Items", Diff: true, Type: "uint64"},
 			},
 		},
 		"unfetched": mp.Graphs{
 			Label: (labelPrefix + " Unfetched"),
 			Unit:  "integer",
 			Metrics: [](mp.Metrics){
 				mp.Metrics{Name: "expired_unfetched", Label: "Expired unfetched", Diff: true, Type: "uint64"},
-				mp.Metrics{Name: "evicted_unfetched", Label: "Evicted unfetched", Diff: true, Type: "uint64"},
```
Comment (astj) [inline mackerel-plugin-memcached/memcached.go]: Why is this metric removed?

Comment (ando-masaki) [inline mackerel-plugin-memcached/memcached.go]: I'm soory. I forgot to revert to the original.

**Comment by ando-masaki** [comment]
I close this pull request because following pull request is similar to my pull request and feel better than mine.
https://github.com/mackerelio/mackerel-agent-plugins/pull/388

**Comment by ando-masaki** [comment]
I close this pull request because following pull request is similar to my pull request and feel better than mine.
#388

**Pull request closed without merging**